packages/types/src/generic/PortableRegistry/PortableRegistry.ts

```diff
@@ -33,6 +33,7 @@ const PATHS_PRIMITIVE = splitNamespace([
   'pallet_democracy::vote::Vote',
   'pallet_identity::types::Data',
   // these are well-known types with additional encoding
+  'core_primitives::EthereumAddress',
```

Comment:
```suggestion
  'account::AccountId20',
```

this seems aligned with the Moonbeam PR, please double-check.

Reply:
As per https://github.com/PureStake/moonbeam/pull/926#discussion_r739500374

Reply:
Going to merge as-is, with this change to make the Monday release. If it changes, we need an update.